src/main/java/techcourse/myblog/DemoApplicationRunner.java

```diff
@@ -1,27 +1,29 @@
-package techcourse.myblog;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.ApplicationArguments;
-import org.springframework.boot.ApplicationRunner;
-import org.springframework.context.annotation.Profile;
-import org.springframework.stereotype.Component;
-import techcourse.myblog.domain.User;
-import techcourse.myblog.domain.repository.UserRepository;
-import techcourse.myblog.support.encrytor.EncryptHelper;
-
-@Component
-@Profile("!test")
-public class DemoApplicationRunner implements ApplicationRunner {
-
-    @Autowired
-    private UserRepository userRepository;
-
-    @Autowired
-    private EncryptHelper encryptHelper;
-
-    @Override
-    public void run(ApplicationArguments args) throws Exception {
-        User user = new User("BROWN", "user@naver.com", encryptHelper.encrypt("Password123!"));
-        userRepository.save(user);
-    }
-}
+package techcourse.myblog;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import techcourse.myblog.domain.User;
+import techcourse.myblog.domain.repository.UserRepository;
+import techcourse.myblog.support.encrytor.EncryptHelper;
+
+@Component
+@Profile("!test")

```

Comment:
```suggestion
@Profile("test")
```
왜 느낌표가 붙어 있는 것이지요..?